### ISSUE TITLE
Get-Module -ListAvailable prints errors from Format file

### ISSUE THREAD

**Issue by felixfbecker** [PowerShell/PowerShell#12896]
## Steps to reproduce

```powershell
Get-Module -ListAvailable
```

## Expected behavior

No errors

## Actual behavior

Output:

```none
> Get-Module -ListAvailable


    Directory: /Users/felix/.local/share/powershell/Modules

ModuleType Version    PreRelease Name                                PSEdition ExportedCommands
---------- -------    ---------- ----                                --------- ----------------
Script     1.0.5      #ERR       Base64                              Desk      {ConvertTo-Base64String, ConvertFrom-Base64String}
Failed to evaluate expression "
                            if ($_.PrivateData -and $_.PrivateData.PSData)
                            {
                                    $_.PrivateData.PSData.PreRelease
                            }".


Script     1.3.1      #ERR       Configuration                       Desk      {Import-Configuration, Export-Configuration, Get-StoragePath, Add-MetadataConverter…}
Failed to evaluate expression "
                            if ($_.PrivateData -and $_.PrivateData.PSData)
                            {
                                    $_.PrivateData.PSData.PreRelease
                            }".


Script     0.1.0      #ERR       GcloudTabComplete                   Core,Desk Add-GcloudToProfile
Failed to evaluate expression "
                            if ($_.PrivateData -and $_.PrivateData.PSData)
                            {
                                    $_.PrivateData.PSData.PreRelease
                            }".


Script     7.0.1      #ERR       ImportExcel                         Desk      {Add-ConditionalFormatting, Add-ExcelChart, Add-ExcelDataValidationRule, Add-ExcelName…}
Failed to evaluate expression "
                            if ($_.PrivateData -and $_.PrivateData.PSData)
                            {
                                    $_.PrivateData.PSData.PreRelease
                            }".


Binary     0.4.1      #ERR       Microsoft.PowerShell.ConsoleGuiToo… Core      {Out-ConsoleGridView, ocgv}
Failed to evaluate expression "
                            if ($_.PrivateData -and $_.PrivateData.PSData)
                            {
                                    $_.PrivateData.PSData.PreRelease
                            }".


Script     0.2.0      #ERR       Microsoft.PowerShell.GraphicalTools Core      {Out-GridView, ogv}
Failed to evaluate expression "
                            if ($_.PrivateData -and $_.PrivateData.PSData)
                            {
                                    $_.PrivateData.PSData.PreRelease
                            }".


Script     2.4.0      #ERR       nvm                                 Desk      {Set-NodeVersion, Install-NodeVersion, Remove-NodeVersion, Get-NodeVersions…}
Failed to evaluate expression "
                            if ($_.PrivateData -and $_.PrivateData.PSData)
                            {
                                    $_.PrivateData.PSData.PreRelease
                            }".


Script     4.9.0                 Pester                              Desk      {Describe, Context, It, Should…}
Script     1.2.3      #ERR       posh-dotnet                         Desk
Failed to evaluate expression "
                            if ($_.PrivateData -and $_.PrivateData.PSData)
                            {
                                    $_.PrivateData.PSData.PreRelease
                            }".


Script     0.4.1                 powershell-yaml                     Desk      {ConvertTo-Yaml, ConvertFrom-Yaml, cfy, cty}
Script     1.3.0      #ERR       PSBuildkite                         Desk      {Get-BuildkiteBuild, Get-BuildkiteCruiseControlFeedUrl, Get-BuildkiteJobLog, Get-BuildkiteOrganization…}
Failed to evaluate expression "
                            if ($_.PrivateData -and $_.PrivateData.PSData)
                            {
                                    $_.PrivateData.PSData.PreRelease
                            }".


Script     0.0.2      #ERR       PSDefaults                          Desk      {Get-DefaultsDomain, Get-DefaultsValue, Set-DefaultsValue, Remove-DefaultsDomain…}
Failed to evaluate expression "
                            if ($_.PrivateData -and $_.PrivateData.PSData)
                            {
                                    $_.PrivateData.PSData.PreRelease
                            }".


Script     1.6.8      #ERR       PSFolderSize                        Desk      {Get-FileReport, Get-FolderSize}
Failed to evaluate expression "
                            if ($_.PrivateData -and $_.PrivateData.PSData)
                            {
                                    $_.PrivateData.PSData.PreRelease
                            }".


Script     0.15.178   #ERR       PSGitHub                            Desk      {Get-GitHubRateLimit, Set-GitHubToken, Get-GitHubUser, Update-GitHubUser…}
Failed to evaluate expression "
                            if ($_.PrivateData -and $_.PrivateData.PSData)
                            {
                                    $_.PrivateData.PSData.PreRelease
                            }".


Script     2.36.2     #ERR       PSGSuite                            Desk      {Add-GSCalendarEventReminder, Add-GSCalendarNotification, Add-GSCalendarSubscription, Add-GSChatButton…}
Failed to evaluate expression "
                            if ($_.PrivateData -and $_.PrivateData.PSData)
                            {
                                    $_.PrivateData.PSData.PreRelease
                            }".


Binary     0.12.0     #ERR       PSKubectl                           Core      {Compare-KubeResource, ConvertFrom-KubeYaml, ConvertTo-KubeYaml, Get-KubeConfig…}
Failed to evaluate expression "
                            if ($_.PrivateData -and $_.PrivateData.PSData)
                            {
                                    $_.PrivateData.PSData.PreRelease
                            }".


Script     0.0.8      #ERR       PSmacOS                             Core      {Get-Clipboard, Set-Clipboard, Out-GridView, Show-MessageBox}
Failed to evaluate expression "
                            if ($_.PrivateData -and $_.PrivateData.PSData)
                            {
                                    $_.PrivateData.PSData.PreRelease
                            }".


Script     0.1.1      #ERR       psmath                              Desk
Failed to evaluate expression "
                            if ($_.PrivateData -and $_.PrivateData.PSData)
                            {
                                    $_.PrivateData.PSData.PreRelease
                            }".


Script     1.0.0      #ERR       PSOpen                              Desk      Open-Item
Failed to evaluate expression "
                            if ($_.PrivateData -and $_.PrivateData.PSData)
                            {
                                    $_.PrivateData.PSData.PreRelease
                            }".


Script     2.1.0      beta1      PSReadLine                          Desk      {Get-PSReadLineKeyHandler, Set-PSReadLineKeyHandler, Remove-PSReadLineKeyHandler, Get-PSReadLineOption…}
Script     1.18.3     #ERR       PSScriptAnalyzer                    Desk      {Get-ScriptAnalyzerRule, Invoke-ScriptAnalyzer, Invoke-Formatter}
Failed to evaluate expression "
                            if ($_.PrivateData -and $_.PrivateData.PSData)
                            {
                                    $_.PrivateData.PSData.PreRelease
                            }".


Script     1.0.1      #ERR       PSSourcegraph                       Core      {Get-Repository, Get-Hover, Get-Definition, Get-Reference…}
Failed to evaluate expression "
                            if ($_.PrivateData -and $_.PrivateData.PSData)
                            {
                                    $_.PrivateData.PSData.PreRelease
                            }".


Script     0.0.1      #ERR       PSTrackingIssue                     Desk      Get-WorkStatusUpdate
Failed to evaluate expression "
                            if ($_.PrivateData -and $_.PrivateData.PSData)
                            {
                                    $_.PrivateData.PSData.PreRelease
                            }".


Script     1.2.2      #ERR       PSTravis                            Desk      {Add-TravisEnvironmentVariable, Enable-TravisRepository, Get-TravisCruiseControlFeedUrl, Get-TravisEnvironmen…
Failed to evaluate expression "
                            if ($_.PrivateData -and $_.PrivateData.PSData)
                            {
                                    $_.PrivateData.PSData.PreRelease
                            }".


Script     0.0.1      #ERR       yarn-completion                     Desk
Failed to evaluate expression "
                            if ($_.PrivateData -and $_.PrivateData.PSData)
                            {
                                    $_.PrivateData.PSData.PreRelease
                            }".



    Directory: /usr/local/microsoft/powershell/7/Modules

ModuleType Version    PreRelease Name                                PSEdition ExportedCommands
---------- -------    ---------- ----                                --------- ----------------
Manifest   1.2.5                 Microsoft.PowerShell.Archive        Desk      {Compress-Archive, Expand-Archive}
Manifest   7.0.0.0               Microsoft.PowerShell.Host           Core      {Start-Transcript, Stop-Transcript}
Manifest   7.0.0.0               Microsoft.PowerShell.Management     Core      {Add-Content, Clear-Content, Clear-ItemProperty, Join-Path…}
Manifest   7.0.0.0               Microsoft.PowerShell.Security       Core      {Get-Credential, Get-ExecutionPolicy, Set-ExecutionPolicy, ConvertFrom-SecureString…}
Manifest   7.0.0.0    #ERR       Microsoft.PowerShell.Utility        Core      {Export-Alias, Get-Alias, Import-Alias, New-Alias…}
Failed to evaluate expression "
                            if ($_.PrivateData -and $_.PrivateData.PSData)
                            {
                                    $_.PrivateData.PSData.PreRelease
                            }".


Script     1.4.6      #ERR       PackageManagement                   Desk      {Find-Package, Get-Package, Get-PackageProvider, Get-PackageSource…}
Failed to evaluate expression "
                            if ($_.PrivateData -and $_.PrivateData.PSData)
                            {
                                    $_.PrivateData.PSData.PreRelease
                            }".


Script     2.2.4      #ERR       PowerShellGet                       Desk      {Find-Command, Find-DSCResource, Find-Module, Find-RoleCapability…}
Failed to evaluate expression "
                            if ($_.PrivateData -and $_.PrivateData.PSData)
                            {
                                    $_.PrivateData.PSData.PreRelease
                            }".


Script     2.0.5      #ERR       PSDesiredStateConfiguration         Core      {Configuration, New-DscChecksum, Get-DscResource, Invoke-DscResource}
Failed to evaluate expression "
                            if ($_.PrivateData -and $_.PrivateData.PSData)
                            {
                                    $_.PrivateData.PSData.PreRelease
                            }".


Script     2.1.0      beta1      PSReadLine                          Desk      {Get-PSReadLineKeyHandler, Set-PSReadLineKeyHandler, Remove-PSReadLineKeyHandler, Get-PSReadLineOption…}
Binary     2.0.3                 ThreadJob                           Desk      Start-ThreadJob

```

## Environment data

```none
> $PSVersionTable

Name                           Value
----                           -----
PSVersion                      7.0.1
PSEdition                      Core
GitCommitId                    7.0.1
OS                             Darwin 19.5.0 Darwin Kernel Version 19.5.0: Tue May 26 20:41:44 PDT 2020; root:xnu-6153.121.2~2/RELEASE_X86_64
Platform                       Unix
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

**Comment by felixfbecker** [comment]
@iSazonov I think this should be labeled as `Issue-Bug` instead of `Issue-Question`

**Comment by iSazonov** [comment]
Could you please test with latest build? I guess this was fixed in #11943.

**Comment by iSazonov** [comment]
Close as stale issue.
